src/Microsoft.AspNetCore.Server.IntegrationTesting/Common/ANCMVersion.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Server.IntegrationTesting
+{
+    public enum ANCMVersion
+    {
+        AspNetCoreModule,
```

Comment:
V1

Reply:
I want to use the name as the value for the ```modules="..."``` section, hence the value doesn't include V1.